versions.lock

```diff
@@ -415,5 +411,6 @@ org.springframework.boot:spring-boot-starter-jetty:2.3.12.RELEASE (1 constraints
 org.springframework.boot:spring-boot-starter-json:2.3.12.RELEASE (1 constraints: 03178f4e)
 org.springframework.boot:spring-boot-starter-logging:2.3.12.RELEASE (1 constraints: 98154deb)
 org.springframework.boot:spring-boot-starter-web:2.3.12.RELEASE (1 constraints: 1d0d3137)
+org.yaml:snakeyaml:1.26 (1 constraints: 0c13dd1f)
```

Comment:
What still depends on this? Should we figure that out and upgrade

Reply:
`./gradlew dependencies` shows that it's pulled in transitively by the s3mock test dep in the `s3-repository` module.

It's weird that the line was missing prior to this commit, but it seems to be legit.   I know Kevin recently updated some of the gradle plugins we were using for dep management and `check` enforcement; maybe related to that in some way?

Reply:
It existed previously. It moved with the previous commit from test only. This just puts it back.